src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```diff
@@ -5,14 +5,14 @@ admin_features_index:
     _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController::indexAction
     _legacy_controller: AdminFeatures
 #    _legacy_link: AdminFeatures
-
-# admin_features_create:
-#  path: /update-position
-#  methods: [ POST ]
+#
+# admin_features_add:
+#  path: /add
+#  methods: [ GET, POST ]
 #  defaults:
-#    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController::updatePosition
```

Comment:
Hmm, we don't need this action to handle the position update? Or was it a wrong copy-paste? So the position update is actually already handled by legacy code from `HelperList` maybe?

Reply:
FeatureController has already implemented create and edit actions, so im pretty sure this was only mistake and it should point to the create action, rather than to not existing updatePosition. Also, yes, the legacy list helper takes care of positioning.

Reply:
Ok, that makes sense